React/my-app/src/screens/home/App.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from "react";
+import data from "./data.json";
+import "./App.css";
+import defaultImg from "./photos/default.png";
+import { WBOOKS, SELECCIONAR_FILTRO, TITULO, AUTOR } from "./strings.js";
+
+class App extends Component {
```

Comment:
Rename component AND file.
Component should be renamed `HomeContainer´
file should be renamed `index`